src/flows/pipes/QueryBuilder/CardList.tsx

```diff
@@ -114,16 +115,35 @@ const Card: FC<Props> = ({idx}) => {
     })
   }
 
+  const isCompliant =
+    isFlagEnabled('newQueryBuilder') &&
+    idx === 0 &&
+    (card.keys.selected.length === 0 ||
+      (card.keys.selected.length === 1 &&
+        card.keys.selected[0] === '_measurement')) &&
+    card.aggregateFunctionType !== 'group' &&
```

Comment:
why does the aggregateFunctionType matter?

Reply:
most of these are for handling migrations. like if someone put a group on a tag right up front, we still need the frontend to support that

Reply:
So on the topic of migrations, this should be compatible with `/from` dashboards, etc... and all existing notebooks should still work if the flag gets turned on right?

Reply:
yes